data/sles4sap/sap_deployment_automation_framework/SAP_SYSTEM.tfvars

```diff
@@ -498,9 +498,9 @@ dns_a_records_for_secondary_names = true
 # AFS indicates that Azure Files for NFS is used
 # ANF indicates that Azure NetApp Files is used
 # NFS indicates that a custom solution is used for NFS
-NFS_provider = "NFS"
+NFS_provider = "AFS"
```

Comment:
Changing to Azure files for NFS. 
NFS option is only if there is already existing NFS server available.